### DISCUSSION TITLE
Show when repositories were last scanned

### DISCUSSION THREAD

**Discussion by stuft2** [community/community#41750] [Code Security]
### Select Topic Area

Product Feedback

### Body

The code scanning section of the security tab for an organization shows when a repository was last updated but it would be more helpful to show when the repository was last scanned. Right now, the only way we can see that information is by viewing the `Security > Code Scanning` page in each repository. Centralizing this bit of information will help developers within organizations detect how frequent code scanning is occurring and further enable security audits on all the repos within an organization.

**Comment by mikeldking** [comment]
Adding a bit to this, recently we moved our monorepo to use `pnpm` from `npm`. Because of this we are now generating an SBOM and uploading it via the dependency submission API (https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/using-the-dependency-submission-api). We intentionally introduced a regression in security (non production dependancy) to test out the dependabot integration but we are not seeing any dependabot alerts. We see the dependancy graph update but there is no sign of a scan. Would love to have a way to trigger or have visibility into how the scan works, in particular with regards to the uploaded SBOMs.
